pkg/system/phase2_creating.go

```diff
@@ -452,6 +452,20 @@ func (r *Reconciler) setDesiredCoreEnv(c *corev1.Container) {
 				c.Env[j].ValueFrom = nil
 			}
 
+		case "POSTGRES_HOST_RO":
+			if r.shouldReconcileCNPGCluster() {
+				//RO host is not part of the cnpg app secret, so we're getting it here from the service
+				services := &corev1.ServiceList{}
+				cnpg_label, _ := labels.Parse("app=noobaa,cnpg.io/cluster=noobaa-db-pg-cluster")
```

Comment:
don't use hard coded labels. you can use the cluster name in `r.CNPGCluster.Name`.   
We should probably also use `r.NooBaa.Name` for the `app=noobaa` label, even though our code already has it hard-coded in a few places.